lib/cocktail/validation/shift.ex

```diff
@@ -15,6 +15,16 @@ defmodule Cocktail.Validation.Shift do
   def shift_by(amount, type, time, option \\ nil)
   def shift_by(0, _, time, _), do: {:no_change, time}
 
+  def shift_by(amount, type, time, option) when type in [:day, "day", :days, "days"] do
```

Comment:
```suggestion
  def shift_by(amount, :days, time, option) do
```

This could be only `:days` (cf typespec + timex)

Reply:
thank you!